libs/gisday/platform/ngx/core/src/lib/pages/admin/pages/university/admin-add-university/admin-add-university.component.scss

```diff
@@ -0,0 +1,13 @@
+@import 'libs/sass/mixins';
+
+label {
+  &.float {
+    transform: translate3d(0, -2rem, 0);
+  }
+}
+
+#hex-preview {
```

Comment:
Preview doesn't seem to work in the first place?

Reply:
It does, you have to go into the edit -> detail page to see it